stateful-workload-filestore/indexInfo.html

```diff
@@ -22,22 +22,21 @@
     <script src="https://ajax.googleapis.com/ajax/libs/jquery/1.7.1/jquery.min.js" type="text/javascript"></script>
 
     <script>
-        $(function () {
-            $("#includedContent").load("indexData.html", function(){
-                var rows = $('text').length;
-                document.getElementById('totalRows').innerHTML = rows;
-                if (rows > 0){
-                    document.getElementById('latestDate').innerHTML = $('text')[rows-1].innerHTML;
-                    document.getElementById('latestWriter').innerHTML = $('text2')[rows-1].innerHTML;
+        $(function() {
+            $('#includedContent').load('indexData.html', function() {
+                const rows = $('text').length;
+                $('#totalRows').html(rows);
+                if (rows > 0) {
+                    $('#latestDate').html($('text')[rows - 1].innerHTML);
+                    $('#latestWriter').html($('text2')[rows - 1].innerHTML);
                 }
-                var allWriters = $('text2');
+
+                const allWriters = $('text2');
                 const mySet = new Set();
-                
-                for(let i = 0; i < allWriters.length; i++){
+                for (let i = 0; i < allWriters.length; i++) {
                     mySet.add(allWriters[i].innerHTML);
                 }
-
-                document.getElementById('uniqueWriters').innerHTML = mySet.size;
+                $('#uniqueWriters').html(mySet.size);
```

Comment:
**Change 1:** We're now using `$('#my-id')` instead of `document.getElementById('my-id')`.
**Change 2:** We're also using jQuery's [`.html()`](https://api.jquery.com/html/) to set the `innerHTML`.